### PR TITLE
Initial implementation of insertBulk for the telemetry db service

### PR DESCRIPTION
**Pull request by marshall**
This initial version can insert multiple entries with an optional timestamp per-entry, an optional top level timestamp, or using system time (the default behavior). Included an initial unit test

/cc @plauche @CatGarab for 👀  and feedback on the API

### CODE REVIEW

**Comment by marshall** [comment]
[Bulk Inserts for Telem DB service to improve input performance](https://trello.com/c/hoeATk0B/707-bulk-inserts-for-telem-db-service-to-improve-input-performance)

File: apis/telemetry-db-api/src/models.rs [hunk 1]
```diff
@@ -17,19 +17,11 @@
...
+#[derive(Debug, Queryable, Serialize, Deserialize, Insertable)]
+#[table_name = "telemetry"]
 pub struct Entry {
     pub timestamp: f64,
     pub subsystem: String,
     pub parameter: String,
     pub value: String,
 }
-
-#[derive(Insertable)]
```
Comment (plauche) [inline apis/telemetry-db-api/src/models.rs]: :+1: Nice that we could collapse this into one model.

File: services/telemetry-service/src/main.rs [hunk 2]
```diff
@@ -204,6 +204,7 @@
 
 #[macro_use]
 extern crate juniper;
+extern crate time;
```
Comment (CatGarab) [inline services/telemetry-service/src/main.rs]: nit: we're using Rust 2018 edition, which no longer requires that crates be declared like this

File: services/telemetry-service/tests/test_insert.rs [hunk 3]
```diff
@@ -269,3 +269,69 @@ fn test_insert_current_timestamp() {
     let difference = (now - timestamp).abs();
     assert!(difference < 1.0);
 }
+
+#[test]
+fn test_insert_bulk() {
```
Comment (CatGarab) [inline services/telemetry-service/tests/test_insert.rs]: Need more unit tests:
- Inserting with a top-level timestamp
- Inserting where at least one entry has its own timestamp

File: services/telemetry-service/src/main.rs (hunk 2 above)
Comment (marshall) [reply services/telemetry-service/src/main.rs]: I'm elated to find out this is a thing

**Comment by marshall** [comment]
Updated with docs, and an update to the telemetry DB benchmark to include both API and GQL bulk insert.

Benchmark results for me locally on a Macbook Pro 6-core 2.2GHz i7 w/ 32GB RAM:
```
+----------------------------+------------------+----------+
| NAME                       | Avg us per-entry | Total us |
+----------------------------+------------------+----------+
| DB API insert (local)      | 541              | 541815   |
+----------------------------+------------------+----------+
| DB API insert_bulk (local) | 39               | 39005    |
+----------------------------+------------------+----------+
| GQL insert (remote)        | 2570             | 2570391  |
+----------------------------+------------------+----------+
| GQL insertBulk (remote)    | 354              | 354878   |
+----------------------------+------------------+----------+
| UDP insert (remote)        | 69               | 69865    |
+----------------------------+------------------+----------+
```

**Pull request merged**